src/granite/transactions.cr

```diff
@@ -11,6 +13,18 @@ module Granite::Transactions
       instance
     end
 
+    def create!(**args)
+      create!(args.to_h)
+    end
+
+    def create!(args : Hash(Symbol | String, DB::Any) | JSON::Any)
+      instance = create(args)
+
+      raise Granite::Transactions::TransactionFailed.new("Could not create #{self.name}") unless instance.errors.empty?
```

Comment:
This line is very long also I dont think you need to specify `Granite::Transactions::`

Reply:
Yeah, looks like Java :rofl:

Reply:
Ok, i worked on validation helper last time and all methods were one liners so i removed my multiline if statement for this one. But i will change it. I will remove the prefix module names